cpp/benchmarks/tpch/utils.hpp

```diff
@@ -50,15 +139,27 @@ inline auto make_managed_pool()
   return rmm::mr::make_owning_wrapper<rmm::mr::pool_memory_resource>(
     make_managed(), rmm::percent_of_free_device_memory(50));
 }
+inline auto make_prefetch()
+{
+  return rmm::mr::make_owning_wrapper<rmm::mr::prefetch_resource_adaptor>(make_managed());
+}
+inline auto make_prefetch_pool()
+{
+  return rmm::mr::make_owning_wrapper<rmm::mr::prefetch_resource_adaptor>(make_managed_pool());
+}
 inline std::shared_ptr<rmm::mr::device_memory_resource> create_memory_resource(
-  std::string const& mode)
+  std::string const& rmm_mode)
 {
-  if (mode == "cuda") return make_cuda();
-  if (mode == "pool") return make_pool();
-  if (mode == "managed") return make_managed();
-  if (mode == "managed_pool") return make_managed_pool();
-  CUDF_FAIL("Unknown rmm_mode parameter: " + mode +
-            "\nExpecting: cuda, pool, managed, or managed_pool");
+  if (rmm_mode == "cuda") return make_cuda();
+  if (rmm_mode == "async") return make_async();
+  if (rmm_mode == "pool") return make_pool();
+  if (rmm_mode == "managed") return make_managed();
+  if (rmm_mode == "managed_pool") return make_managed_pool();
+  if (rmm_mode == "prefetch") return make_prefetch();
+  if (rmm_mode == "prefetch_pool") return make_prefetch_pool();
+  CUDF_FAIL(
+    "Unknown rmm_mode parameter: " + rmm_mode +
+    "\nExpecting: cuda, async, pool, async_pool, managed, managed_pool, prefetch, prefetch_pool");
```

Comment:
I don't see `async_pool` above.
```suggestion
    "\nExpecting: cuda, async, pool, managed, managed_pool, prefetch, prefetch_pool");
```